sdk/core/azure-core-tracing-opencensus/tests/test_tracing_implementations.py

```diff
@@ -155,12 +155,16 @@ def test_span_kind(self):
             assert wrapped_class.span_instance.span_kind == OpenCensusSpanKind.CLIENT
             assert wrapped_class.kind == SpanKind.CLIENT
 
-            # not supported
-            with pytest.raises(ValueError):
-                wrapped_class.kind = SpanKind.PRODUCER
+            # opencensus doesn't support producer, put client instead
+            wrapped_class.kind = SpanKind.PRODUCER
+            assert wrapped_class.span_instance.span_kind == OpenCensusSpanKind.CLIENT
+            assert wrapped_class.kind == SpanKind.CLIENT
 
-            with pytest.raises(ValueError):
-                wrapped_class.kind = SpanKind.CONSUMER
+            # opencensus doesn't support consumer, put server instead
```

Comment:
consumer is also client kind (this is the call to 'receive' that we don't instrument)
the processing  span is 'server'

Reply:
fixed